Modify uri poi type

azime opened the pull request:
- Ajout de "amenity" et "leisure" dans l'uri du type de poi
- Suppresion des fichiers "pois.cpp" et "pois.h" inutilisables

azime committed:
ed

add "amenity", "leisure" in uri poi_type

azime committed:
ed

ed : delete "pois" files unused

TeXitoi commented on source/ed/ed_reader.cpp:
```diff
@@ -1089,7 +1089,7 @@ void EdReader::fill_graph_bss(navitia::type::Data& data, pqxx::work& work){
                 request += " FROM georef.poi poi, georef.poi_type poi_type";
                 request += " where poi.poi_type_id=poi_type.id";
-                request += " and poi_type.uri = 'poi_type:bicycle_rental'";
+                request += " and poi_type.uri = 'poi_type:amenity:bicycle_rental'";
```
Same change line 1114?

azime committed:
ed

- add "amenity" in uri poi_type
-remove "poi_type:park_ride" and "poi_type:Airesdestationnement"

azime commented:
dans la requête de la fonction "fill_graph_parking", suppression de 'poi_type:park_ride' et 'poi_type:Airesdestationnement' et ajout de 'amenity' dans l'uri de poi_type
